assets/assets.ttslua

```diff
@@ -1,16 +1,16 @@
 g_assets =
 {
-  dir="https://raw.githubusercontent.com/marcpawl/Triumph_TTS/e808443dd8604747f889438a1e0a02a253d4ee07/assets/",
-  counter_background="https://raw.githubusercontent.com/marcpawl/Triumph_TTS/e808443dd8604747f889438a1e0a02a253d4ee07/assets/images/counter_background.png",
-  token_no_camp="https://raw.githubusercontent.com/marcpawl/Triumph_TTS/e808443dd8604747f889438a1e0a02a253d4ee07/assets/images/no_camp_token/no_camp.png",
-  token_charge_through="https://raw.githubusercontent.com/marcpawl/Triumph_TTS/e808443dd8604747f889438a1e0a02a253d4ee07/assets/images/charge_through_token/charge_through.png",
-  token_hold_the_line="https://raw.githubusercontent.com/marcpawl/Triumph_TTS/e808443dd8604747f889438a1e0a02a253d4ee07/assets/images/hold_the_line/hold_the_line.png",
-  prepared_defenses_2d_diffuse_url="https://raw.githubusercontent.com/marcpawl/Triumph_TTS/e808443dd8604747f889438a1e0a02a253d4ee07/assets/prepared_defenses/prepared_defenses_2d.png",
-  prepared_defenses_2d_mesh_url="https://raw.githubusercontent.com/marcpawl/Triumph_TTS/e808443dd8604747f889438a1e0a02a253d4ee07/assets/prepared_defenses/prepared_defenses_2d.obj",
-  flank_march_card_red="https://raw.githubusercontent.com/marcpawl/Triumph_TTS/e808443dd8604747f889438a1e0a02a253d4ee07/assets/flank_march_card/flank_march_card_red.png",
-  flank_march_card_blue="https://raw.githubusercontent.com/marcpawl/Triumph_TTS/e808443dd8604747f889438a1e0a02a253d4ee07/assets/flank_march_card/flank_march_card_blue.png",
-  rock_1_obj="https://raw.githubusercontent.com/marcpawl/Triumph_TTS/e808443dd8604747f889438a1e0a02a253d4ee07/assets/terrain/terrain_rock_grey/terrain_rock_grey_1.obj",
-  rock_2_obj="https://raw.githubusercontent.com/marcpawl/Triumph_TTS/e808443dd8604747f889438a1e0a02a253d4ee07/assets/terrain/terrain_rock_grey/terrain_rock_grey_2.obj",
-  rock_3_obj="https://raw.githubusercontent.com/marcpawl/Triumph_TTS/e808443dd8604747f889438a1e0a02a253d4ee07/assets/terrain/terrain_rock_grey/terrain_rock_grey_3.obj",
-  rock_4_obj="https://raw.githubusercontent.com/marcpawl/Triumph_TTS/e808443dd8604747f889438a1e0a02a253d4ee07/assets/terrain/terrain_rock_grey/terrain_rock_grey_4.obj",
+  dir="file:///D:/Tabletop Simulator Mods/new_triumph_tts/assets/",
```

Comment:
Do not push changes made with clean_save unless they are for remote.  you can do ./clean_save --force-remote